lib/curses_renderer.rb

```diff
@@ -16,35 +17,22 @@ def initialize(board)
     @cpu_player = Cpu.new(board)
   end
 
-  def show_game_over_screen
-    Curses.addstr("\n\nGAME OVER!\n\n")
-    Curses.refresh
-    sleep(3)
-  end
```

Comment:
This is still here, I just moved it lower to sort of reflect a hierarchy of importance within the file.